app/views/claims/support/claims/index.html.erb

```diff
@@ -8,20 +8,40 @@
 
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-one-third">
-      <div class="app-filter">
-        <div class="app-filter__header">
-          <div class="govuk-heading-m">Filter</div>
-        </div>
-      </div>
+      <%= render partial: "filter" %>
     </div>
 
     <div class="govuk-grid-column-two-thirds">
-      <%= render partial: "shared/search_form", locals: {
-        url: claims_support_claims_path,
-        label: { text: t(".search_label"), size: "s" },
-        name: :search,
-        clear_search_url: claims_support_claims_path,
-      } %>
+      <div class="govuk-form-group">
+        <%= form_with(
+          url: claims_support_claims_path,
+          scope: :claims_support_claims_filter_form,
+          class: "search-form",
+          method: :get,
+          data: { turbo: false },
+        ) do |f| %>
```

Comment:
I think it might be easier to manage the filter/search states if this form encompasses both the filters and the search. Otherwise, submitting 1 form would reset the other form, no?

Reply:
I'll try to get both forms working as one

Reply:
It might be a bit strange for the user, as the filters will be applied when you click search or the search would be triggered when you click apply filters

Reply:
Hmm, I'm not sure. I've previously run into a case where users were surprised that their search query wasn't being removed.

https://github.com/DFE-Digital/itt-mentor-services/assets/4231530/ef454d71-8554-48b8-833c-9604473a9b10

Shall we ask Product?

Reply:
Probably ask product, the video example you show is happening in the prototype as well. It's just a side effect of having to click buttons rather than live search. I'll ask

Reply:
Seems like they want search and filtering to be separate https://ukgovernmentdfe.slack.com/archives/C0657JE64HX/p1712929956668779?thread_ts=1712929049.555999&cid=C0657JE64HX